cloud-controller-manager/do/loadbalancers.go

```diff
@@ -40,6 +40,22 @@ const (
 	// used to enable fast retrievals of load-balancers from the API by UUID.
 	annoDOLoadBalancerID = "kubernetes.digitalocean.com/load-balancer-id"
 
+	// annDOLoadBalancerName is the annotation used to specify a name of the load
+	// balancer that is going to be created by the controller.
+	//
+	// The name must not be longer than 255 characters and must adhere to the
+	// following naming convention:
+	// * it must start with an alphanumeric character;
+	// * it must consist of alphanumeric characters or the '.' (dot) or '-'
+	// (dash) characters; except for the final character which must not be '-'
+	// (dash). Violations to any of these rules will lead to reconciliation
+	// errors. If no custom name is given, a default name is chosen consisting of
+	// the character 'a' appended by the Service UID.
+	//
+	// Adding this annotation to an existing service will also result in renaming
+	// an existing LoadBalancer associated with it.
```

Comment:
Can we "merge" this line with the very first sentence of the comment? Right now, the first sentence mentions creates only, and it requires the reader to go through the entire configuration description before realizing that existing LBs are affected as well. I'd prefer that to be more immediately obvious.

Reply:
Sure, makes sense.